tests/Fixtures/Controller/PreCRUDController.php

```diff
@@ -24,27 +24,27 @@
  */
 class PreCRUDController extends CRUDController
 {
-    protected function preCreate(Request $request, $object)
+    protected function preCreate(Request $request, object $object): ?Response
```

Comment:
```suggestion
    protected function preCreate(Request $request, object $object): Response
```

You can restrict (same for others)